### PR TITLE
Get user

### PR DESCRIPTION
**Pull request by gmckz**
Implemented get/users/:id.

Awaiting feedback from Leo / Eddie before approval.

### CODE REVIEW

File: api/spec/controllers/users.spec.js [hunk 1]
```diff
@@ -58,5 +61,74 @@ describe("/users", () => {
...
+  //   await user2.save();
+    
+  //   const userId = user2.id
+  //   //make the GET request to users/:id using the user id above
+  //   let getResponse = await request(app)
+  //     .get(`/users/${userId}`)
+  //     // .set("Authorization", `Bearer ${token}`)
+  //     // .send({user: user2, token: token});
+  //   expect(getResponse.statusCode).toBe(401)
+
```
Comment (FahimIslam2410) [inline api/spec/controllers/users.spec.js:132]: Are these commented out test/code needed or can they be deleted?

**Review by FahimIslam2410 — CHANGES_REQUESTED** [review]

File: api/spec/controllers/users.spec.js (hunk 1 above)
Comment (gmckz) [reply api/spec/controllers/users.spec.js]: commented out tests now deleted

**Review by FahimIslam2410 — APPROVED** [review]

File: api/spec/controllers/users.spec.js (hunk 1 above)
Comment (EdwardAndress) [reply api/spec/controllers/users.spec.js]: Good spot @FahimIslam2410 and nice work acting on the feedback quickly @gmckz !

**Comment by EdwardAndress** [comment]
Should there also be a test for what happens when the token is not present?

**Comment by EdwardAndress** [comment]
Do you need to use tokenChecker when handling this new request and, if so, how can you do that without forcing someone to sign in before signing up? :)

**Pull request merged**